src/applications/vaos/covid-19-vaccine/index.jsx

```diff
@@ -33,20 +32,22 @@ import {
   selectCanUseVaccineFlow,
   selectDirectScheduleSettingsStatus,
 } from '../appointment-list/redux/selectors';
+import { fetchDirectScheduleSettings } from '../appointment-list/redux/actions';
 
-export function NewBookingSection({
-  canUseVaccineFlow,
-  featureCovid19Vaccine,
-  directScheduleSettingsStatus,
-  fetchDirectScheduleSettings,
-}) {
+export default function NewBookingSection() {
   const match = useRouteMatch();
   const history = useHistory();
   const location = useLocation();
+  const dispatch = useDispatch();
+  const canUseVaccineFlow = useSelector(selectCanUseVaccineFlow);
+  const featureCovid19Vaccine = useSelector(selectFeatureCovid19Vaccine);
+  const directScheduleSettingsStatus = useSelector(
+    selectDirectScheduleSettingsStatus,
+  );
 
   useEffect(() => {
     if (directScheduleSettingsStatus === FETCH_STATUS.notStarted) {
-      fetchDirectScheduleSettings();
+      dispatch(fetchDirectScheduleSettings);
```

Comment:
Missing `()` at the end of `fetchDirectScheduleSettings`

Reply:
@jbalboni good catch but still didn't fix the default error. I am assuming when I pulled the connect it broke something I am not seeing